activities/communication/using-chat.md

```diff
@@ -0,0 +1,66 @@
+---
+type: guide
+---
+
+# How to use [chat.publiccode.net](https://chat.publiccode.net/)
+
+We use Mattermost, hosted at chat.publiccode.net for informal chat purposes.
+
+Mattermost lets us create multiple teams with different members. We only have 1 team right now: our staff team.
+
+## Staff team
+
+Discussions on chat.publiccode.net are:
+
+* timely
+* optional
+* secret
+
+If something must be known by all staff, please send an email.
+
+### Channels
+
+We have channels that reflect our work, plus some crosscutting topics.
+
+Feel free to create new channels as you discover you need them.
+
+When creating a new channel for an [activity documented on About](https://about.publiccode.net/activities/), please standardize the label between chat.publiccode.net and about.publiccode.net - what we talk about is the most honest reflection of what we're actually doing.
+
+## Getting started
+
+Take these steps to join the staff team.
+
+1. The operations coordinator will send you an invitation.
+
+2. Set up your account. Please use the same name from as your email address as your username so it’s easy to see who’s who. Also, please upload an avatar (👹) and fill in your full name.
+
+3. Join relevant channels!
+
+4. Add integrations. We encourage you to
+
+```/github connect```
+
+You may also want to add
+
+ ```/github settings notifications off```
+
+ and
+
+```/github settings reminders off```
+
+if you don't want Mattermost to send you emails about Github issues or pull requests you're subscribed to.
+
+5. Say hi 👋👋.
```

Comment:
This is not valid markdown and will create a section with separate lists of 1 item with different starting numbers.

I suggest

```suggestion
1. The operations coordinator will send you an invitation.
2. Set up your account. 
  Please use the same name from as your email address as your username so it’s easy to see who’s who.
  Also, please upload an avatar (👹) and fill in your full name.
3. Join relevant channels!
4. Add integrations.
  We encourage you to `/github connect`.
  You may also want to add `/github settings notifications off` and `/github settings reminders off` if you don't want Mattermost to send you emails about Github issues or pull requests you're subscribed to.
5. Say hi 👋👋.
```

Reply:
And when I run these changes locally I get no objections from the tests.